HLTriggerOffline/Btag/test/config.ini

```diff
@@ -8,42 +8,42 @@ files= root://cms-xrd-global.cern.ch//store/relval/CMSSW_8_1_0_pre7/RelValTTbar_
 
 ;Paths names and btag variables that you want to monitor
 [btag]
-HLT_PFMET120_:
+HLT_PFMET120_PFMHT120_IDTight_v:
 	hltCombinedSecondaryVertexBJetTagsCalo
 
-HLT_HT300PT30_QuadJet_75_60_45_40_v:
+HLT_PFHT300PT30_QuadPFJet_75_60_45_40_v:
 	hltCombinedSecondaryVertexBJetTagsCalo
 
-HLT_PFHT380_SixJet32_DoubleBTagCSV_:
+HLT_PFHT380_SixPFJet32_DoublePFBTagCSV_:
 	hltCombinedSecondaryVertexBJetTagsCalo
 
 HLT_Ele28_eta2p1_WPTight_Gsf_HT150_:
 	hltCombinedSecondaryVertexBJetTagsPF
 
-HLT_IsoMu24_eta2p1_:
+HLT_IsoMu24_eta2p1_v:
 	hltCombinedSecondaryVertexBJetTagsPF
 
 ;Paths names and vertex variables that you want to monitor
 [vertex]
 
-HLT_PFMET120_:
+HLT_PFMET120_PFMHT120_IDTight_v:
 	hltFastPrimaryVertex
 	hltFastPVPixelVertices
 	hltVerticesL3
 
-HLT_HT300PT30_QuadJet_75_60_45_40_v:
+HLT_PFHT300PT30_QuadPFJet_75_60_45_40_v:
 	hltFastPrimaryVertex
 	hltFastPVPixelVertices
 	hltVerticesL3
 
-HLT_PFHT380_SixJet32_DoubleBTagCSV_:
+HLT_PFHT380_SixPFJet32_DoublePFBTagCSV_:
```

Comment:
shouldn't it be
`HLT_PFHT380_SixPFJet32_DoublePFBTagCSV_v`

Reply:
@mtosi in this case it should be HLT_PFHT380_SixPFJet32_DoublePFBTagCSV_2p2_v , but in the previous version it was HLT_PFHT380_SixJet32_DoubleBTagCSV_p075_v, so keeping in mind possible future changes I would leave this path as is (without 2p2_v)